suite-common/wallet-utils/src/__tests__/validation.test.ts

```diff
@@ -79,6 +79,10 @@ describe('validation', () => {
         expect(isAddressValid('0xE37c0D48d68da5c5b14E5c1a9f1CFE802776D9FF0', 'trop')).toEqual(
             false,
         );
+        expect(isAddressValid('0x7de62F23453E9230cC038390901A9A0130105A3c', 'tgor')).toEqual(true);
```

Comment:
in fact, this throws error, so the tests here are not very useful :D see followup issue https://github.com/trezor/trezor-suite/issues/6599

Reply:
I see, I removed that test, we can address it in the follow up issue. Thanks.